src/components/CheckEditor/CheckProbes/PrivateProbesAlert.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { Alert, LinkButton, Stack } from '@grafana/ui';
+
+import { ROUTES } from 'types';
+import { getRoute } from 'components/Routing.utils';
+
+export const PrivateProbesAlert = () => {
+  const [display, setDisplay] = useState(true);
```

Comment:
Let's keep a localStorage memory of their dismissal of this banner. Some people just won't be interested in setting up private probes so I don't see the point in us reminding them for a feature they aren't interested in.

Reply:
Good idea 👍